### DISCUSSION TITLE
How can I overwrite PON WAN setting created by the OLT ?

### DISCUSSION THREAD

**Discussion by TDAD68** [Anime4000/RTL960x#158] [Q&A]
Hello.  

Currently I have DFP-34X-2C2 and it's connected to internet. 

However, I have one issue that I can't change IP protcol that on the PON WAN setting due to this setting is created by the OLT. 

My ISP provide IPv6 as well but somehow they didn't set IPv6 as a IP protocol. 

if anyone know the solution then much appreciated.

![Screenshot_2023-05-12-00-38-05-692-edit_org.mozilla.fenix.jpg](https://github.com/Anime4000/RTL960x/assets/17574993/13952087-b0f2-4872-9e84-e2f8d1b22827)

**Comment by Anime4000** [comment]
try use SFU type firmware (bridge) and VLAN it on your router
